app/common/utils/coingecko.ts

```diff
@@ -27,7 +27,7 @@ export async function getPrice({
   }).toString();
 
   try {
-    const data = await fetch(url, { signal: abortSignal }).then(r => r.json());
+    const data = await fetch(url, { cache: 'default', signal: abortSignal }).then(r => r.json());
```

Comment:
This might fix [task](https://app.clickup.com/t/86952fvh5), but I've not reproduced coingeko error, because for me cache was working as intended 🤷‍♂️